Cargo.toml

```
@@ -1,5 +1,6 @@
 [workspace]
 members = [
+  "fuzz",
```

I think that the convention is to have the directory name match the package name.

```suggestion
  "neqo-fuzz",
```

---

I undid this and instead changed the directory name to `fuzz`. Otherwise one has to pass the `--fuzz-dir neqo-fuzz` argument on each invocation of the fuzzer (because it assumes the default is `fuzz`), which is annoying.